docs/extensibility.md

```diff
@@ -27,129 +27,26 @@ wp.blocks.registerBlockType( 'mytheme/red-block', {
 
 If you want to learn more about block creation, The [Blocks Tutorial](./blocks) is the best place to start.
 
+## Extending Blocks
 
-## Removing Blocks
+It is also possible to modify the behaviour of the existing blocks or even remove them completely using filters.
```

Comment:
Maybe we should use behavior instead (American English). I checked and our code base uses both but the number of occurrences of "behavior" is much higher.